client/my-sites/earn/memberships/products.jsx

```diff
@@ -322,6 +343,109 @@ class MembershipsProductsSection extends Component {
 						) }
 					</FormToggle>
 				</FormFieldset>
+			</>
+		);
+	}
+
+	renderEmailTab() {
+		const { translate } = this.props;
+		const { editedPostsEmail, editedCustomConfirmationMessage } = this.state;
+		return (
+			<>
+				<FormFieldset>
+					<h6 className="memberships__dialog-form-header">{ translate( 'Posts via email' ) }</h6>
+					<p>
+						{ translate(
+							'Allow members of this recurring payment plan to opt into receiving new posts via email.'
+						) }
+						<a href="/TODO">{ translate( 'Learn more' ) }.</a>
```

Comment:
@Automattic/earn any idea about where this "Learn more" link do should point to?

Reply:
This would be the page: https://wordpress.com/support/wordpress-editor/blocks/recurring-payments-button/#connecting-stripe, but it needs some new content.

Reply:
We can probably use `InlineSupportLink` then if we're pointing at a support page.

Reply:
f1adc05